tests/cframework/tests_plugin.h

```diff
@@ -19,9 +19,9 @@
 	succeed_if (output_warnings (errorKey), "warnings in kdbOpen for plugin " NAME);                                                   \
 	succeed_if (output_error (errorKey), "error in kdbOpen for plugin " NAME);                                                         \
 	keyDel (errorKey);                                                                                                                 \
-	exit_if_fail (plugin != 0, "could not open " NAME " plugin");
+	exit_if_fail (plugin != 0, "could not open " NAME " plugin")
```

Comment:
A better solution is to wrap everything with a do while loop, then it does not matter if the user of the macro adds `;`.

It would also be nice if the duplication of PLUGIN_OPEN can be avoided.